types/node/globals.d.ts

```diff
@@ -100,6 +100,16 @@ interface BigInt64Array extends RelativeIndexable<bigint> {}
 interface BigUint64Array extends RelativeIndexable<bigint> {}
 //#endregion ArrayLike.at() end
 
+/**
+ * @since 17.0.0
```

Comment:
Nit: the other `@since` tags appear to use `v17.0.0`

Reply:
Good pickup thanks, will update this instance and then fix [these other instances](https://cs.github.com/?scopeName=All+repos&scope=&q=repo%3ADefinitelyTyped%2FDefinitelyTyped+%2F%40since+%5Cd%2F+path%3A%2F%5Etypes%5C%2Fnode%5C%2F%2F) in a seperate PR 👍